bindings/dart/lib/src/common/reranker/ai.dart

```diff
@@ -65,6 +65,13 @@ class XaynAi {
   /// Retrieves the analytics which were collected in the penultimate reranking.
   Analytics? analytics() => throw UnsupportedError('Unsupported platform.');
 
+  /// Serializes the synchronizable data of the reranker.
+  Uint8List syncdataBytes() => throw UnsupportedError('Unsupported platform.');
+
+  /// Synchronizes the internal data of the reranker with another.
+  void synchronize([Uint8List? serialized]) =>
```

Comment:
since this argument is marked as optional, does it make sense to call the method without any arguments? i would think that we could make this required like in the reranker of `xayn_ai`:
```suggestion
  void synchronize(Uint8List serialized) =>
```
(the c ffi still needs to take an `Option<&CBytes>`, as this represents any thin pointer to a `CBytes` fat pointer, where the former theoretically could also be null.)